.github/workflows/automated-tests.yml

```diff
@@ -24,9 +24,9 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: "1.19" # The Go version to download (if necessary) and use.
-
       - name: Unit, integration and difference tests
         run: go test ./...
-
       - name: E2E tests
         run: make test-e2e-short
+      - name: Proto Check
```

Comment:
Lmk if yall want this in it's own workflow .yml, I just tagged onto the end og automated-tests.yml for now